README.md

```diff
@@ -10,7 +10,7 @@ The gem is require'd in the rspec loading script, and will take effect from ther
 
 ```ruby
 describe "GET index" do
-  it "is successful", fixture: "widgets/index.json" do
+  it "is successful", rcv: {fixture: "spec/fixtures/widgets/index.json"} do
```

Comment:
There is a base_path which I encourage setting to avoid saying `spec/fixtures`. It does default to nil, so having this explicit in the docs is worthwhile.